components/credit-card/FinanceMultipleChoice.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from "react"
+
+interface FinanceMultipleChoiceProps {
+    question:string;
+    options: string[];
+} 
+
+export const FinanceMultipleChoice = ({ question, options }: FinanceMultipleChoiceProps) => {
```

Comment:
where is this component even used?

Reply:
It is being used in CCDCMulti.tsx but I renamed it to reflect the component better